OpenMTR/FileHandler.cs

```diff
@@ -0,0 +1,267 @@
+                metadataPath = standardizePath(value);
+            }
+        }
+
+
+
+        /// <summary>
+        /// Return the current data struct, read only
+        /// </summary>
+        public Files Data
+        {
+            get
+            {
+                return currentData;
+            }
+        }
+
+
+
+        /// <summary>
+        /// Return the full path of the current image
+        /// </summary>
+        public string ImageFile
+        {
+            get
+            {
+                return Dir + filenames[pos];
+            }
+        }
+
+
+
+        /// <summary>
+        /// Return the full path of the current metadata file
+        /// </summary>
+        public string MetadataFile
+        {
+            get
+            { // technically one line, broken out for readability
+                return
+                    MetadataDir +
+                    Regex.Replace(
+                        filenames[pos], extensions, "txt"
+                        );
+            }
+        }
+
+
+
+        //================================ STATIC METHODS ===========================================
+
+
+
+        /// <summary>
+        /// Converts a file to a bitmap, direct copy of Matt's method in the canny filter program
+        /// </summary>
+        /// <param name="fileName">The filename</param>
+        /// <returns>A bitmap</returns>
+        public static Bitmap ConvertToBitmap(string fileName)
+        {
+            Bitmap bitmap;
+            using (Stream bmpStream = File.Open(fileName, FileMode.Open))
+            {
+                Image image = Image.FromStream(bmpStream);
+
+                bitmap = new Bitmap(image);
+
+            }
+            return bitmap;
+        }
+
+
+
+        /// <summary>
+        /// Replace forward slashes with backslashes and add a trailing backslash if not present
+        /// </summary>
+        /// <param name="input"></param>
+        /// <returns>A standardized path</returns>
+        public static string standardizePath(string input)
+        {
+            string output = input.Replace('/','\\').Trim();
+
+            if (output[output.Length - 1] == '\\') return output;
+            return output + "\\";
+        }
+
+
+
+        /// <summary>
+        /// Returns the integer contained in a plain text file
+        /// </summary>
+        /// <param name="fileName">The name of file with complete path</param>
+        /// <returns>Whatever integer was found in the file</returns>
+        public static int ConvertToInt(string fileName, bool noFail)
+        {
+            string number = File.ReadAllText(fileName);
+
+            number = number.Replace("[\n\r]", "").Trim(); // I was going to trim anyway, so I might as well get rid of new lines while I'm at it
+
+            if (noFail) // If the file contains some kind of comment or other silliness this can be used to purge every non-numeric character

```

Comment:
>The way I see it, it's either we assume that all files have only numbers in them and are proven wrong later, or we sanitize all metadata.
That was also my reasoning. It doesn't do anything for now, but if it turns out we need to sanitize the function to do so is already here.
In retrospect, this would be more gracefully and efficiently handled by a try/catch block. Do you think that one should be implemented or just get rid of this altogether?